UQ-eLIPSE/crucible-components
# Story/vethub 15

Rendering question options

In the `question-data.json` file I've added some sample options to render.

To test:
- Run `yarn dev` and then navigate to `https://localhost:5173` 
- Verify the options matches with the `question-data.json` options.
- Run `yarn test` to run the updated tests.
